.circleci/config.yml

```diff
@@ -15,21 +33,27 @@ jobs:
       - checkout
       - restore_cache:
           keys: 
+            - maven-repo-v1-{{ checksum "pom.xml" }}-{{ .Environment.DEPENDENCIES }}
             - maven-repo-v1-{{ checksum "pom.xml" }}
             - maven-repo-v1-
       - run:
           name: Update permissions
           command: sudo chown -R circleci /usr/local/lib/node_modules
+      - install-dependencies
       - run:
           name: Build
           command: |
             java -version
             mvn -v
-            mvn -B clean install
+            if [[ "$DEPENDENCIES" == "snapshot" ]]; then
+              mvn -B clean install -Pcloud,fedDev
```

Comment:
What is this `fedDev` profile?

Reply:
@dplaton added this as part of enabling https://github.com/adobe/aem-cif-guides-venia/tree/main/ui.frontend#developing-with-aem-core-cif-react-components.

Reply:
@cjelger it actually comes from https://github.com/adobe/aem-project-archetype/blob/master/src/main/archetype/ui.frontend.general/pom.xml#L85